crates/fuel-core/src/state/in_memory.rs

```diff
@@ -1,2 +1,2 @@
+// TODO: Move the implementation from the module to here.
```

Comment:
Is this part of a future PR?

Reply:
I don't want to move it in this PR not to mix the changes and provide a clear diff of what actually was changed in the `in_memory` module. In can be done by someone who will notice this todo=)